pkg/cmd/config.go

```diff
@@ -501,7 +509,10 @@ kind: KubeOneCluster
 name: {{ .ClusterName }}
 
 versions:
-  kubernetes: "{{ .KubernetesVersion }}"
+ kubernetes: "{{ .KubernetesVersion }}"
+
+kubeletConfiguration:
+ containerLogMaxSize: "{{ .ContainerLogMaxSize }}"
```

Comment:
since there should be no v1beta1 API change, whole diff in this file should also be gone

Reply:
This is not relevant anymore since the example config uses the v1beta2 API already.